Realign Administer menu, move User menu down

bittner opened the pull request:
This PR fixes a usability issue with the "double menu", one on top of the other, on the right-hand side of the top menu.

Now,

- the _Administer_ menu is aligned normally with all the other menu items of the top menu,
- the _User (Account)_ menu is where any user would expect it -- on the right-hand side, and
- the _Notifications_ area is on top of the _User (Account)_ menu -- a natural place to draw attention.

![foreman-administer-user-menu-fixed](https://user-images.githubusercontent.com/665072/31508084-475e8eb0-af7d-11e7-83a2-697122de5b59.png)

Note: The screenshot was made from an account with restricted permissions (i.e. not Administrator permissions).

More Potential (optional)
-------------------

What could be done in addition is make the font a bit bigger for the user name (in the Account menu caption). I refrained from doing now that to keep the PR small.

Would be happy if this PR were accepted, thanks!

bittner committed:
Realign Administer menu, move User menu down

waldenraines commented:
@bittner thanks for the PR!  I'm wondering if you have looked at https://github.com/theforeman/foreman/pull/4794 (which changes the navigation to a vertical layout) and if that PR addresses your concerns.

bittner commented:
@waldenraines the solution in PR #4794 is indeed awesome! And it perfectly addresses the concerns I was trying to solve with the little change of mine. Thanks!

I'm looking forward to the new UI with the vertical menu layout!
